README.md

```diff
@@ -296,7 +305,7 @@ This test framework abstracts away all that complexity, allowing for a smooth TD
 
 **Couldn't we just use the MVP pattern with clear interfaces at the boundaries?**  
 _Yes we could... but would we?  
-Let's be pragmatic, with that kind of project we're developing for our home, and we're a team of one.
+Let's be pragmatic, with what kind of project we're developing for our home, and we're a team of one.
```

Comment:
With:
`Let's be pragmatic, with that kind of project we're developing for our home, and we're a team of one.`
I meant:
`Let's be pragmatic, with THIS kind of project we're developing for our home, and we're a team of one.`

Would you mind updating `that` -> `this` instead of `what`. I agree the original formulation was confusing. If you think with `this` it's still confusing we can find an alternative version. But `what` changes the intent of the sentence.

Thanks for pointing it out.